client/my-sites/customer-home/cards/actions/wp-for-teams-quick-links-compact/index.jsx

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import {
+	getSiteFrontPage,
+	getCustomizerUrl,
+	getSiteOption,
+	isNewSite,
+} from 'state/sites/selectors';
+import { getSelectedEditor } from 'state/selectors/get-selected-editor';
+import isSiteUsingFullSiteEditing from 'state/selectors/is-site-using-full-site-editing';
+import { getGSuiteSupportedDomains } from 'lib/gsuite';
+import { getDomainsBySiteId } from 'state/sites/domains/selectors';
+import { navigate } from 'state/ui/actions';
+import {
+	bumpStat,
+	composeAnalytics,
+	recordTracksEvent,
+	withAnalytics,
+} from 'state/analytics/actions';
+import ActionBox from 'my-sites/customer-home/cards/actions/quick-links/action-box';
+
+/**
+ * Style dependencies
+ */
+import 'my-sites/customer-home/cards/actions/quick-links-compact/style.scss';
+
+export const QuickLinks = ( {
+	customizeUrl,
+	isStaticHomePage,
+	showCustomizer,
+	menusUrl,
+	editHomepageAction,
+	writePostAction,
+	addPageAction,
+	manageCommentsAction,
+	trackEditMenusAction,
+	trackCustomizeThemeAction,
+} ) => {
+	const translate = useTranslate();
+
+	const quickLinks = (
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Should we maybe follow the className naming here so folks can search for the component a bit more easily?

Reply:
Yeah, sure. Main reason why didn't want to follow the classname namespace pattern was to reuse the regular quick links styles, but we can actually use 2 classes here 🙂

Reply:
Done in a62ba00